subprojects/core/src/test/groovy/org/gradle/internal/progress/DefaultBuildOperationExecutorTest.groovy

```diff
@@ -645,4 +647,10 @@ class DefaultBuildOperationExecutorTest extends ConcurrentSpec {
 
         void run(BuildOperationContext buildOperationContext) {}
     }
+
+    ParallelExecutionManager parallelExecutionManager() {
```

Comment:
⭕️ There are lots of these snippets throughout the tests now, which probably means this deserves a dedicated stub impl.